File: Capítulo_3/ECMAScript 2015 – os métodos find e findIndex/findIndex/Código em Python/findIndex.py
# Array de número naturais
numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]

def multiple_of_13(element):
    return element % 13 == 0

# Em Python você pode utilizar a função index com filter diretamente em uma lista para encontrar o índice do primeiro elemento que satisfaça uma condição específica. 

# Encontrar o índice do primeiro elemento que satisfaça a condição
try:
    index = numbers.index(next(filter(multiple_of_13, numbers)))
except ValueError:
    index = -1

print(index)

File: Capítulo_3/ECMAScript 2015 – os métodos find e findIndex/findIndex/findIndex.js
// Array de número naturais
let numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];

function multipleOf13(element, index, array) {
    return(element % 13 === 0)
}

// findIndex -> Busca um elemento no array, dada uma condição desejada (função de callback), e devolve o índice do elemento caso
console.log(numbers.findIndex(multipleOf13)); // 12